source/mainnet/net/guides/tokens.rst

```diff
@@ -0,0 +1,80 @@
+.. include:: ../../variables.rst
+.. _tokens:
+
+========================
+Tokens in the wallet
+========================
+
+You can add, inspect, and send tokens in the |bw|.
+
+.. dropdown:: |bw|
+
+    To access tokens and token transactions, click Tokens |token-bw|. Tabs appear for tokens.
+
+    .. dropdown:: Add tokens
+
+        #. Click **Manage**.
+
+            .. image:: ../images/browser-wallet/manage-tokens.png
+                :width: 50%
+
+        #. Enter a smart contract address on which to look for tokens. Click **Choose contract**. If you get an error that says **Invalid contract address** this means that the smart contract doesn’t exist, the smart contract doesn’t live up to the CIS-2 standard, or the smart contract doesn’t contain any tokens.
+
+            .. image:: ../images/browser-wallet/add-token-contract.png
+                :width: 50%
+
+        #. Select the token(s) you want to add. You can click on a token to see more information about it and then click **Back to list** to return to the list to select tokens. Click **Update tokens** once you have made your selection(s).
```

Comment:
You can't click back to list, but rather click the close icon to close the detail view and go back to viewing the list of tokens.